tasks/clientlibs.js

```diff
@@ -220,16 +224,17 @@ module.exports = function (grunt) {
 							clientLibCSS = getIncludeString(clientLibName, 'css');
 
 							// Sort the array into dependency order
-							clientLibCSSObj = performSort(clientLibCSSObj);
+							clientLibCSSOrderObj = performSort(clientLibCSSObj);						
```

Comment:
Changing this to clientLibCssOrderObj makes me think it's the order. Why add "Order"?

Reply:
That's true, the output is the correct order css/js files according depends attribute.